docs/setup-cluster/k8s/resource-caps.rst

```diff
@@ -0,0 +1,135 @@
+.. _k8s-resource-caps:
+
+#####################################
+ Manage Workspace Namespace Bindings
+#####################################
+
+.. note::
+
+   Auto creating namespaces and managing resource quotas in Determined require the Determined
+   Enterprise Edition.
+
+Determined makes it easier to manage resource quotas by handling them directly, so you don't need to
```

Comment:
@hkumar92 should we warn the users about [this](https://hpe-aiatscale.slack.com/archives/C06RZSV56FM/p1721077493638219) case here?

Reply:
Yes we should let them know especially of potential footguns we've tested and aware of

Reply:
@tara-det-ai can we add a warning that if a job is pending due to lack of resources (not resource quota), and they decide to change the resource quota, the job will get scheduled despite any restricting resource quota

Reply:
thank you for the summarized description of the use case. i've added an "important" admonition.